### PR TITLE
Add unsubs

### PR DESCRIPTION
**Pull request by codyss**
Updates to add unsubs and bounces events
Also, include selected in catalog

### CODE REVIEW

File: tap_listrak/__init__.py [hunk 1]
```diff
@@ -22,6 +22,7 @@ def discover(ctx):
     for tap_stream_id in schemas.stream_ids:
         schema = Schema.from_dict(schemas.load_schema(tap_stream_id),
                                   inclusion="automatic")
+        schema.selected = True if tap_stream_id in ['lists', 'messages'] else False
```
Comment (cmcarthur) [inline tap_listrak/__init__.py:25]: why add this line?

**Review by cmcarthur — COMMENTED** [review]
the changes look reasonable. were you able to run the tap with your credentials?

File: tap_listrak/__init__.py (hunk 1 above)
Comment (codyss) [reply tap_listrak/__init__.py:25]: So lists and messages should be on by default, per Buck's README, right? So this turns them on and then everything else is false unless turned on. It took me a little while to realize that was what was breaking it when I was running locally. 

Before selected wasn't there at in the catalog.

**Comment by codyss** [comment]
Yeah, I was able to run successfully.

File: tap_listrak/__init__.py (hunk 1 above)
Comment (cmcarthur) [reply tap_listrak/__init__.py:25]: oh, I understand -- this is in discovery. this makes sense.

**Review by cmcarthur — APPROVED** [review]

**Pull request merged**